definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js

```diff
@@ -35,8 +37,8 @@ declare module 'lolex' {
     runMicrotasks(): void;
     runToFrame(): void;
     runToLast(): void;
-    setSystemTime(now?: Date): void;
-    uninstall(): void;
+    setSystemTime(now?: number | Date): void;
+    uninstall(): Object[];
```

Comment:
https://github.com/sinonjs/lolex/blob/d4c2c053c8dce7d8b9e6727ac75275e5e32bae2b/src/lolex-src.js#L426-L432

Reply:
(might wanna type the `timers` object better)